build-release.sh

```diff
@@ -312,6 +308,20 @@ if [ "${build_classical}" == "1" ]; then
   rm -rf macos_template.app
   sign_macos_template ${templatesdir} 0
 
+  ## Steam (Classical) ##
+
+  if [ -d out/windows/steam ]; then
+    cp out/windows/steam/godot.windows.editor.x86_64.exe ${steamdir}/godot.windows.opt.tools.64.exe
+    cp out/windows/steam/godot.windows.editor.x86_32.exe ${steamdir}/godot.windows.opt.tools.32.exe
+    sign_windows ${steamdir}/godot.windows.opt.tools.64.exe
+    sign_windows ${steamdir}/godot.windows.opt.tools.32.exe
+    unzip ${reldir}/${godot_basename}_linux.x86_64.zip -d ${steamdir}/
+    unzip ${reldir}/${godot_basename}_linux.x86_32.zip -d ${steamdir}/
+    mv ${steamdir}/{${godot_basename}_linux.x86_64,godot.x11.opt.tools.64}
+    mv ${steamdir}/{${godot_basename}_linux.x86_32,godot.x11.opt.tools.32}
+    unzip ${reldir}/${godot_basename}_macos.universal -d ${steamdir}/
+  fi
```

Comment:
Here we're preparing the Steam binaries to be signed (for Windows, it's a custom Steam-specific build) and with the expected names (for Linux). This is just to make it easier to copy the binaries in `publish-release.sh` directly.